src/ConversationLearner.ts

```diff
@@ -90,10 +90,11 @@ export class ConversationLearner {
         this.clRunner.AddCallback(callback)
     }
 
-    /** Define an Callback that will be called after Entity Detection
+    /**
+     * Define an Callback that will be called after Entity Detection
      * @param target Callback of the form (text: string, memoryManager: ClientMemoryManager) => Promise<void>
      */
-    public EntityDetectionCallback(target: EntityDetectionCallback) {
+    set EntityDetectionCallback(target: EntityDetectionCallback) {
```

Comment:
Would also like to change this to `camelCase` since it's already breaking change but all of the other parts of the SDK are Pascal / C#-y so I left it.